tools/torch_quant/tests/test_module_copy_and_replace.py

```diff
@@ -22,13 +23,28 @@ def test_root(self) -> None:
         copied = copy_and_replace(model, mapping)
         self.assertIs(copied, mapping[''].gm)
 
-    @unittest.skip('not implemented')
     def test_replace_single(self) -> None:
-        ...
+        model = SimpleModule()
+        dummy_input = torch.randn((1, 2, 5, 5))
+        include_name = 'conv'
+        module_filter = ModuleFilter(include_names=[include_name])
+        mapping = fx_trace(model, module_filter)
+        copied = copy_and_replace(model, mapping)
+        self.assertEqual(len(mapping), 1)
+        self.assertIs(getattr(copied, include_name), mapping[include_name].gm)
+        self.assertTrue(torch.equal(model(dummy_input), copied(dummy_input)))
 
-    @unittest.skip('not implemented')
     def test_replace_multiple(self) -> None:
-        ...
+        model = SimpleModule()
+        dummy_input = torch.randn((1, 2, 5, 5))
+        include_names = ['conv', 'sub', 'linear']
+        module_filter = ModuleFilter(include_names=include_names)
+        mapping = fx_trace(model, module_filter)
+        copied = copy_and_replace(model, mapping)
+        self.assertEqual(len(mapping), len(include_names))
+        for name in include_names:
+            self.assertIs(getattr(copied, name), mapping[name].gm)
+        self.assertTrue(torch.equal(model(dummy_input), copied(dummy_input)))
```

Comment:
`test_replace_single` and `test_replace_multiple` can be merged into one test function with parameterized. Or, just keep `test_replace_multiple` (rename to `test_replace`), I can't remember why we put two not-implemented test functions here 😂

Reply:
Done, merged into one test function with parameterized.
_(Maybe to notice the scenarios needed to be coped with.)_